open-grant-proposals/open-proposal-miner-analyzer.md

```diff
@@ -0,0 +1,173 @@
+# Open Grant Proposal: `Filecoin miner analyzer and stats`
+
+**Name of Project:** Filecoin miner stats
+
+**Proposal Category:** `app-dev`
+
+**Proposer:** `nodefactoryio`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** Yes!
+
+# Project Description
+
+When a a mining node is installed and stared, a miner has a hard time of tracking what's happening with the node and gaining insights into the node's performance. This is almost always the case as such software is installed on the server and requires time for connecting to it and checking if everything is all right.
+
+The solution we propose is a responsive web application with a notification system that gives full insight into node's status. The web app would give analysis to miner about the node performance, earnings, uptime and available disk space. In case there is a required action to be made i.e. a node goes down or software update is required, email notification would be sent to the user. Anything that can possibly help miners to improve their doing.
+
+## Value
+
+This is an added value for the miners and has no risk of failure for the Filecoin ecosystem. As miners have the clear incentive to make money on the provided service, we want to make sure they keep up with it and get the right statistics on their performance, how much money they are making and how to possibly improve that.
+
+If we get this right, there is a possibility to expand this depending on the future development of Filecoin, for example it's reputation system. 
+
+## Deliverables
+
+The project will be considered as finished when the following is delivered:
+
+* Clear project specification
+* UI/UX design for both mobile and web version
+* Frontend application (React)
+* Backend application (Node.Js)
+* Light daemon app for setup and uptime tracking (Go)
+* User onboarding and project documentation
+
+The web app should be able to provide to user the following functionalities:
+* registration and login with email,
+* adding nodes to be tracked (generation of URL for our daemon app),
+* email notifications on required action
+
+and insights into: 
+* node connection status,
+* calculated profit,
+    * earnings through block rewards
+    * retrieval earnings
+* history of past and active deals with their status,
+    * inspection of deal's state
+    * payment redeemal status
+* utilization and disk space occupancy,
+* status of pledged collateral (past slashings).
+
+Note that this functionalities may slightly change as we are open to adapting to user's needs or Filecoin network changes discovered during the development process. 
+
+## Development Roadmap
+
+### Milestone 1
```

Comment:
Would this project make sense to start after testnet? Our APIs, protocol, and implementations will change significantly between now and then.

Reply:
It is certainly something that we wanted to go through together with you to know what changes we can expect and plan accordingly. However, we think that milestone 1 (and even part of milestone 2) is not necessarily affected by those changes as it is more about all the use cases and UI/UX design.

Reply:
Understood, thanks for clarifying.